neptune-python-utils/neptune_python_utils/endpoints.py

```diff
@@ -188,7 +189,12 @@ def __init__(self, neptune_endpoint=None, neptune_port=None, region_name=None, c
         else:
             session = boto3.session.Session()
             self.region = session.region_name
-            
+
+        if sts_endpoint:
+            self.sts_endpoint = sts_endpoint
+        else:
+            self.sts_endpoint = 'https://sts.amazonaws.com'
```

Comment:
Correct, that'd be a safer thing to do. Since underlying default url may change.

Reply:
can we modify the code to always use the regional endpoint? and get rid of the parameter

Reply:
As per our offline discussion, I've made the relevant changes and also updated the readme with explanation of the scenario in which it will be critically useful.